adr/0001-community-discussions.adoc

```diff
@@ -0,0 +1,70 @@
+- its hard to monitor as contributor who wants to help answer/ask questions.
+
+How can we improve this situaton and enable the broader community to more easily ask questions and find answers - without it all be relying on just a few Quarkus core contributors?
+
+== Scenarios (optional)
+
+User wants to locate a answer to a question - zulip chats does not show up in google search; stackoverflow might but might not have an answer.
+
+Contributor want to arrange or attend an event around Quarkus - where do he post/look for info on that ?
+
+Contributor/committer are trying to follow new questions and issues - what does he need to track to do that ? today it is alot and it is all different notification sytems and mainly its a big "firehose" with no good ability to filter out what you don't need.
+
+User looking for a job involving Quarkus - where do he go look ?
+
+
+////
+ as well as a lot of social platform presence https://twitter.com/quarkusio/[twitter], https://www.facebook.com/quarkusio/[facebook], https://www.linkedin.com/groups/13789086/[linkedin], https://www.youtube.com/quarkusio[youtube] and https://www.linkedin.com/groups/13789086/[reddit].
+////
+
+== Considered options
+
+* stackoverflow will not go away - thus for this current proposal it does not include Q/A section.
+* "I can't track more places" is a very valid worry; the advantage of discussions is that it shows up in your normal github notifcations which you can mute, filter,etc. like anything else on Github. Thus it gives you more control rather than just adding more noise.
+
+=== quarkusio-community mailing list
+Not proposed since it is just another waterhose of info with no good ability to ignore/consume as needed/wanted.
+
+=== enable discussions on github.com/quarkusio/quarkus-insights
+Considered as quarkus insights is an existing "community" and let us have separate permissons for moderators/triage but some suggested it might be hard to find.
+
+=== enable discussions on github.com/quarkusio/community
+Downside is that labels etc. would need to be kept in sync and that searches in github.com/quarkusio would not show up community/discussions.
+It could show up if searched on github.com/quarkus level though.
+Could be considered as allow us to have separate permissions for moderator/triagers separate from commit access - need to check if https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-permission-levels-for-an-organization[github docs] on permissions levels work for us.
+
+== Decision
+
+Enable GitHub discussions feature on `quarkusio/quarkusio` with the following initial categories:
+
+- Announcements (post only by admins)
+- Quarkus Comunity (post by anyone, general discussions)
+- Quarkus Insights Episodes (show notes and offline comments)
+- Quarkus World Tour and other Events (Setup and announce of interest or other events)
+- Jobs and Opportunities (post by anyone)
```

Comment:
One thing I would like would be a place where people can present themselves if they wish. E.g. one thread per person with the name of the person and they describe what they are doing with Quarkus.

I'm not sure people would want to do that but when I interact with people I often ask myself about the context or what they're doing and things like that.

Again no idea if anyone would participate to that.

Reply:
+1; some will. What should we call it ? or maybe just a post with certain label in quarkus community ?

Reply:
+1, sounds like a good opportunity to know who is using Quarkus in their jobs

Reply:
Did it go away? I don't see "Introductions" in the list?

Reply:
+1 to introduction and yes it's not on the list

Reply:
git push mishab - its there now.